### PR TITLE
ICP-8341 Update setLevel to follow capability spec

### PR DESCRIPTION
**Pull request by dkirker**
Right now, I have all of the matches included. We can more than likely remove a few, but it is probably safe to just include them all.

### CODE REVIEW

**Comment by tpmanley** [comment]
I believe Keen needs to be updated too

File: devicetypes/capabilities/switch-level-capability.src/switch-level-capability.groovy [hunk 1]
```diff
@@ -71,7 +71,7 @@ def off() {
 	'off'
 }
 
-def setLevel(value) {
+def setLevel(value, rate = null) {
```
Comment (tpmanley) [inline devicetypes/capabilities/switch-level-capability.src/switch-level-capability.groovy:74]: I'm not familiar with how these files are used so I'm not sure if this is OK or not

**Review by tpmanley — APPROVED** [review]

File: devicetypes/capabilities/switch-level-capability.src/switch-level-capability.groovy (hunk 1 above)
Comment (dkirker) [reply devicetypes/capabilities/switch-level-capability.src/switch-level-capability.groovy:74]: I *think* they are capability examples. @bflorian could possibly confirm?

**Comment by andrewsayre** [comment]
This fixes #3986

File: devicetypes/capabilities/switch-level-capability.src/switch-level-capability.groovy (hunk 1 above)
Comment (andrewsayre) [reply devicetypes/capabilities/switch-level-capability.src/switch-level-capability.groovy:74]: I understand them to be examples as well.

**Comment by andrewsayre** [comment]
Any update on when this will go in?

**Comment by dkirker** [comment]
@tpmanley Let's go over a QA plan tomorrow? I am sure this is a safe merge, but I want to make sure that it does get some level of verification.

**Review by rappleg — APPROVED** [review]

**Comment by dkirker** [comment]
Once @workingmonk does the deploy tomorrow, we can go ahead and merge this. (Rebasing to master and merging now would be ugly.)

**Comment by workingmonk** [comment]
@dkirker let me know after the deploy tomm

**Pull request merged**